simulation_parameters/SimulationParameters.cs

```diff
@@ -85,6 +85,14 @@ private SimulationParameters()
 
     public NameGenerator NameGenerator { get; }
 
+    public override void _Notification(int what)
+    {
+        if (what == NotificationTranslationChanged)
+        {
+            ApplyTranslations();
```

Comment:
Doesn't options menu already directly call this when changing the translations?
I think one of these 2 approaches should be removed.

Reply:
I tried with using only the call in Settings.cs and the it seems the compounds and processes names (specifically on the charts and organelle tooltips) doesn't properly change like it stuck one language behind, maybe godot notifications are executed earlier than script functions? Either way, I guess I'll keep the notifications approach.

Reply:
But if just this notifications doesn't work, then shouldn't notifications be removed from SImulationParameters, as it is already specifically handled by the game options to notify the parameters?

Reply:
I mean to say the notification in SimulationParameters works by itself and the apply translations call in the settings doesn't work properly on its own.

Reply:
Ah okay, so you meant to remove the call from options to SimulationParameters? That's fine as well.

Reply:
Yeah, that's what I meant.